packages/cli/lib/sandbox-create.js

```diff
@@ -0,0 +1,333 @@
+const Spinnies = require('spinnies');
+const {
+  sandboxNamePrompt,
+  sandboxTypePrompt,
+} = require('./prompts/sandboxesPrompt');
+const {
+  sandboxTypeMap,
+  getSandboxLimit,
+  getHasSandboxesByType,
+  saveSandboxToConfig,
+  sandboxApiTypeMap,
+  getSandboxTypeAsString,
+  getAccountName,
+  STANDARD_SANDBOX,
+  DEVELOPER_SANDBOX,
+} = require('./sandboxes');
+const { i18n } = require('@hubspot/cli-lib/lib/lang');
+const { logger } = require('@hubspot/cli-lib/logger');
+const {
+  debugErrorAndContext,
+  logErrorInstance,
+} = require('@hubspot/cli-lib/errorHandlers/standardErrors');
+const { trackCommandUsage } = require('./usageTracking');
+const {
+  isMissingScopeError,
+  isSpecifiedError,
+} = require('@hubspot/cli-lib/errorHandlers/apiErrors');
+const { getHubSpotWebsiteOrigin } = require('@hubspot/cli-lib/lib/urls');
+const { ENVIRONMENTS } = require('@hubspot/cli-lib/lib/constants');
+const { getEnv, getAccountConfig, getConfig } = require('@hubspot/cli-lib');
+const { createSandbox } = require('@hubspot/cli-lib/sandboxes');
+const { promptUser } = require('./prompts/promptUtils');
+const { syncSandbox } = require('./sandbox-sync');
+const {
+  setAsDefaultAccountPrompt,
+} = require('./prompts/setAsDefaultAccountPrompt');
+const { updateDefaultAccount } = require('@hubspot/cli-lib/lib/config');
+
+const i18nKey = 'cli.commands.sandbox.subcommands.create';
+
+/**
+ * @param {String} name - Name of sandbox
+ * @param {String} type - Sandbox type to be created (standard/developer)
+ * @param {Object} accountConfig - Account config of parent portal
+ * @param {String} env - Environment (QA/Prod)
+ * @param {Boolean} allowEarlyTermination - Option to allow a keypress to terminate early
+ * @param {Boolean} skipDefaultAccountPrompt - Option to skip default account prompt and auto set new sandbox account as default
+ * @returns {Object} Object containing sandboxConfigName string and sandbox instance from API
+ */
+const buildSandbox = async ({
+  name,
+  type,
+  accountConfig,
+  env,
+  allowEarlyTermination = true,
+  skipDefaultAccountPrompt = false,
+  skipSyncStandardSandbox = false,
+  force = false,
```

Comment:
Moved all sandbox creation logic into this util with the following params to tap into